monkey/monkey_island/cc/services/agent_binary_service/flask_resources/agent_binary_masque.py

```diff
@@ -0,0 +1,36 @@
+import logging
+from http import HTTPStatus
+from json import JSONDecodeError
+
+from flask import request
+from flask_security import auth_token_required, roles_accepted
+
+from monkey_island.cc.flask_utils import AbstractResource
+from monkey_island.cc.services.authentication_service import AccountRole
+
+from .. import IAgentBinaryService
+
+logger = logging.getLogger(__name__)
+
+
+class AgentBinaryMasque(AbstractResource):
+    urls = ["/api/agent-binary-masque"]
```

Comment:
As discussed in Webex, this needs to be independently configurable for Windows and Linux.